src/metaschema/readme.md

```diff
@@ -37,18 +37,18 @@ Within it `valid-values`, a `value` element's `@name` attribute assigns the perm
 
 ## Metaschema modeling
 
-In the case of field and flag objects, the modeling constraints to be imposed by the result schemas (either XSD or JSON Schema) over the data set, can be determined on the basis of how they are described. Assembly definitions, however, permit not only flags to be assigned to assemblies of the defined type; additionally, they contain a `model` element for a *mode declaration*. This declaration names the subcomponents to be permitted (in documents valid to the target schemas) within any assembly of the type being defined.
+In the case of field and flag objects, the modeling constraints to be imposed by the result schemas (either XSD or JSON Schema) over the data set can be determined on the basis of how they are described. Assembly definitions, however, permit not only flags to be assigned to assemblies of the defined type; additionally, they contain a `model` element for a *model declaration*. This declaration names the subcomponents to be permitted (in documents valid to the target schemas) within any assembly of the type being defined.
 
-Five elements are used within `model` to define permissible contents of assemblies (elements or objects) being defined. Each of these represents a different object type. Flags are not assigned via `model` but directly in the definition; for the model, we can choose either singles or plurals of named fields or assemblies (i.e., a binary choice between cardinality constraints to be applied). This gives us four choices; additionally, we have the opportunity to use an element `prose`, once inside any assembly's model.
+Five elements are used within `model` to define permissible contents of assemblies (elements or objects) being defined. Each of these represents a different object type. Flags are not assigned via `model` but directly in the definition; for the model, we can choose either singles or plurals of named fields or assemblies (i.e., a binary choice between cardinality constraints to be applied). This gives us four choices; additionally, we have the opportunity to use an element `prose`, at most once inside any assembly's model.
```

Comment:
The reason I added "at most" was that on first reading of "once inside," I didn't know if the word once was meant in a more colloquial way, as in "once you are inside."

Alternatively, you could end the sentence with something like "... an element `prose` in any assembly's model." and rely on line 44 to indicate the multiplicity.

Reply:
I think your impressions are very good, but I have a larger piece of feedback that will obsolete this feedback. It is good, but I want to go even leaner. 😄